en/flight_modes/rtl.md

```diff
@@ -0,0 +1,45 @@
+# RETURN TO LAUNCH (RTL) Flight Mode
+
+The RTL flight mode causes the vehicle to return to its home position where it may then either wait (hover or circle) or land.
+
+> **Note** 
+>  * This mode requires GPS.
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+
+The specific behaviour for each vehicle type is described below.
+
+## Multi-Copter (MC)
+
+A copter/multi-rotor will first ascend to the `RTL_RETURN_ALT` altitude and then fly to the home position in a straight line (if already above `RTL_RETURN_ALT` it will return at its current altitude).
+
+When it arrives at the home/launch position it will rapidly descend to the `RTL_DESCEND_ALT` altitude. It will then hover for the period defined in `RTL_LAND_DELAY` before landing.
+
+> **Note** The `RTL_LAND_DELAY` is provided to allow time for landing gear to be deployed (this is triggered automatically). By default this period is short so that the vehicle will simply slow and then land immediately. The parameter can also be set so that the vehicle will hover indefinitely.
+
+The RTL behaviour can be configured using the parameters below.
+
+
+Parameter | Description
+--- | ---
+[RTL_RETURN_ALT](../advanced_config/parameter_reference.md#RTL_RETURN_ALT) | Minimum RTL return altitude in meters (default: 60m). The vehicle will ascend to this altitude before returning. If already above this value the vehicle will return at its current altitude.
+[RTL_DESCEND_ALT](../advanced_config/parameter_reference.md#RTL_DESCEND_ALT) | Altitude at which the vehicle will slow or stop its initial descent (default: 30m)
+[RTL_LAND_DELAY](../advanced_config/parameter_reference.md#RTL_LAND_DELAY) | Time to hover at `RTL_DESCEND_ALT` before landing (default: 0.5s). If set to -1 the system will loiter at `RTL_DESCEND_ALT` rather than landing.
+[RTL_MIN_DIST](../advanced_config/parameter_reference.md#RTL_MIN_DIST) | Minimum distance to trigger rising to a safe altitude. If the system is horizontally closer than this distance to home it will land straight on home instead of raising to the return altitude first.
+
+
+## Fixed Wing (FW)

```

Comment:
@dagar My understanding is that RTL can land or circle. If it does land (because you have set RTL_LAND_DELAY>0) then it will do so using flare etc (the same land as you get in mission)
- is that correct?

I think we should separately document this landing because it is used in mission as well. Then link to it. OK?